Services/EventRaisingResourceManager.cs

```diff
@@ -0,0 +1,57 @@
+﻿using IDeliverable.Widgets.Events;
+using Orchard.Environment.Extensions;
+using Orchard.UI.Resources;
+using System.Collections.Generic;
+using Autofac.Features.Metadata;
+
+namespace IDeliverable.Widgets.Services
+{
+    [OrchardFeature("IDeliverable.Widgets.OutputCache")]
+    [OrchardSuppressDependency("Orchard.UI.Resources.ResourceManager")]
+    public class EventRaisingResourceManager : ResourceManager
+    {
+        private readonly IResourceManagerEvents _resourceManagerEvents;
```

Comment:
All this class does is wraps (ideal use case for decorators) the default impl in order to raise `IResourceManagerEvents` events.

If the default impl was updated to raise events then this imple would no longer be required.